dawn/CMakeLists.txt

```
@@ -132,10 +77,8 @@ if(DAWN_PYTHON)
 endif()
 
 # Build dawn
-include_directories(${CMAKE_SOURCE_DIR}/src)
-include_directories(${CMAKE_BINARY_DIR}/src)
-include_directories(SYSTEM ${CMAKE_SOURCE_DIR}/test/utils/googletest/include)
-include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/src/dawn/Support/External)
+include_directories(${DAWN_SOURCE_DIR}/src)
```

If you do as suggested above, then this should be
```suggestion
include_directories(${PROJECT_SOURCE_DIR}/src)
```
and there is no need for `DAWN_SOURCE_DIR`, `GTCLANG_SOURCE_DIR` etc.